### DISCUSSION TITLE
Project status

### DISCUSSION THREAD

**Discussion by Abhishek-Sathe-Tafe** [ZiyangWang001007081/Tafe230243#7] [General]
Hi team,

I've finished developing the mortgage feature branch and have committed the changes to the feature branch. I'm now ready to merge my feature with the development branch. Please let me know if I have got a go-ahead from you to do so?

**Comment by ZiyangWang001007081** [comment]
Hi Abhi,

Well done! Please carry on with the merging. I'll perform my merging after you pushed the latest version to the cloud.

Cheers

**Comment by Abhishek-Sathe-Tafe** [comment]
Hi team,

Please note that the mortgage branch merge has been completed. Please pull the latest revision for development branch before you execute your merge.

Thanks!
Abhi
